src/axom/sidre/examples/sidre_data_vs_metadata.cpp

```diff
@@ -225,12 +230,14 @@ int main(int argc, char* argv[])
   // Example 3: One-to-many Buffer to View relationships (view copy)
   // -----------------------------------------------------------------------
   //
-  // Create a buffer with a double array of length dat_size and initialize its
-  // data. Attach the buffer to two views, with data descriptions that vary in
-  // offset. Verify that the data associated with each view is correct.
+  // Create a copy of a view in one group in another group. Verify that the
+  // views are the same as well the fact that they share the same data.
```

Comment:
```suggestion
  // views are the same as well the fact that they share the same data.
  // views have the same length, offset, and stride, and that they share the same data.
```
Original was too vague.

Reply:
I changed it to "views share the same data (i.e., they have the same base address, length, offset, and stride)."

Reply:
And somehow 😒 I added the line, instead of replacing it.   OK, so the sentence is "Verify that the views share the same data (i.e. they have the same base address, length, offset, and stride)."  That will work well.  I do suggest removing the comma after `i.e.`.   Thanks.